lsstswBuild.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
I am not.   Error exit is the equivalent of catching or letting an exception bubble up in python.  I don't think we need to comment every try block.